.eslintrc

```diff
@@ -75,7 +75,8 @@
     "env": {
         "es6": true,
         "node": true,
-        "browser": true
+        "browser": true,
+        "jest": true
```

Comment:
This will allow as to drop the:
```
/* global expect: false */
/* global it: false */
/* global describe: false */
```
comments that are at the top of all of our spec files.